apps/round/src/app/getType.ts

```diff
@@ -20,8 +20,8 @@ export async function getTypeByName(name: string): Promise<TypePokemon> {
 async function buildTypePokemon(pokemon: any): Promise<TypePokemon> {
   const type: TypePokemon = {
     label: pokemon.name as LabelTypePokemon,
-    faiblesses: pokemon.damage_relations.double_damage_from.map(e => e.name) as LabelTypePokemon[],
-    resistances: pokemon.damage_relations.half_damage_from.map(e => e.name) as LabelTypePokemon[],
+    faiblesses: pokemon.damage_relations.double_damage_from.map((e: any) => e.name) as LabelTypePokemon[],
```

Comment:
TypeScript's any type is a super and subtype of all other types, and allows dereferencing all properties. As such, any is dangerous - it can mask severe programming errors, and its use undermines the value of having static types in the first place.

Consider not to use any. In circumstances where you want to use any, consider one of:
[Provide a more specific type](https://google.github.io/styleguide/tsguide.html#any-specific)
[Use unknown](https://google.github.io/styleguide/tsguide.html#any-unknown)
[Suppress the lint warning and document why](https://google.github.io/styleguide/tsguide.html#any-suppress)

Reply:
On ne fait pas d'omelette sans casser des œufs